src/workflows-app/components/LeftNavigationPanel.ts

```diff
@@ -0,0 +1,156 @@
+import _ from 'lodash/fp';
+import { useState } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
+import Collapse from 'src/components/Collapse';
+import { Clickable } from 'src/components/common';
+import { centeredSpinner, icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import { getConfig } from 'src/libs/config';
+import * as Style from 'src/libs/style';
+import HelpfulLinksBox from 'src/workflows-app/components/HelpfulLinksBox';
+import { SubmissionHistory } from 'src/workflows-app/SubmissionHistory';
+
+const subHeadersMap = {
+  'workspace-workflows': 'Workflows in this workspace',
+  'submission-history': 'Submission history',
+} as const;
+
+const findAndAddSubheadersMap = {
+  'featured-workflows': 'Featured workflows', // Update in follow-up tickets
+  'import-workflow': 'Import a workflow', // Update in follow-up tickets
+};
+
+export const LeftNavigationPanel = ({ name, namespace, workspace, loading }) => {
+  const [selectedSubHeader, setSelectedSubHeader] = useState<string>('workspace-workflows');
```

Comment:
Can we type guard this subheader since it should be one of a few limited values? Something like
```typescript
type SubheaderKey = keyof typeof subHeadersMap | keyof typeof findAndAddSubheadersMap

... useState<SubheaderKey> ...
```
This would ensure that we don't ever accidentally set the wrong value, as well as provide exhaustive checking in our switch cases, especially if we add/remove/modify these in the future.